search_similar_entities/searchSimilarEntitiesWidget.html

```diff
@@ -0,0 +1,25 @@
+    <script>
```

Comment:
Does this contain ERB code (looks like it)?  Should use .erb instead of .html as file extension